challenge/files/09_slices.toml

```diff
@@ -0,0 +1,56 @@
+name = "Slice Append Behavior"
+category = "wtf"
+default_code_snippet = """
+func main() {
+	var x []int
+	x = append(x, 0)
+	x = append(x, 1)
+	x = append(x, 2)
+
+	fmt.Println(append(append(x, 3), append(x, 4)...))
```

Comment:
I think it's better to divide it into 3 lines.

```
y3 := append(x, 3)
y4 := append(x, 4)

fmt.Println(append(y3, y4...))
```

So, we can add question like "What causes unexpected output?" with `y4` line answer